src/Illuminate/Container/Container.php

```diff
@@ -547,26 +547,27 @@ public function factory($abstract)
     }
 
     /**
-     * Resolve the given type with the given parameter overrides.
+     * An alias function name for make().
      *
      * @param  string  $abstract
      * @param  array  $parameters
      * @return mixed
      */
-    public function makeWith($abstract, array $parameters)
+    public function makeWith($abstract, array $parameters = [])
```

Comment:
Although we could - I was just thinking about minimising unnecessary breaking changes for 5.5 or onwards.

For 2-3 lines of code it seems simple to keep - and I've added a framework test to ensure it works as intended (so wont be lost in future refactors).

Up to Taylor - happy to remove if needed.